dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/api/RequestApi.kt

```diff
@@ -0,0 +1,95 @@
+package org.dataland.datalandcommunitymanager.api
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import jakarta.validation.Valid
+import org.dataland.datalandbackend.openApiClient.model.DataTypeEnum
+import org.dataland.datalandcommunitymanager.entities.DataRequestEntity
+import org.dataland.datalandcommunitymanager.model.dataRequest.AggregatedDataRequest
+import org.dataland.datalandcommunitymanager.model.dataRequest.BulkDataRequest
+import org.dataland.datalandcommunitymanager.model.dataRequest.BulkDataRequestResponse
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+
+/**
+ * Defines the restful dataland-community-manager API regarding.
+ */
+@RequestMapping("/requests")
+@SecurityRequirement(name = "default-bearer-auth")
+@SecurityRequirement(name = "default-oauth")
+interface RequestApi {
+
+    /**
+     * A method to post a bulk request to Dataland.
+     * @param bulkDataRequest includes necessary info for the bulk request
+     * @return response after posting a bulk data request to Dataland
+     *
+     */
+    @Operation(
+        summary = "Send a bulk request",
+        description = "A bulk of data requests for specific frameworks and companies is being sent.",
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Successfully processed a bulk of data requests."),
+        ],
+    )
+    @PostMapping(
+        produces = ["application/json"],
+        consumes = ["application/json"],
+    )
+    @PreAuthorize("hasRole('ROLE_USER')")
+    fun postBulkDataRequest(
+        @Valid @RequestBody
+        bulkDataRequest: BulkDataRequest,
+    ):
+        ResponseEntity<BulkDataRequestResponse>
+
+    /** A method to get all data requests of a specific user.
+     * @return all data requests of the user in a list
+     */
+    @Operation(
```

Comment:
Verify that the swagger ui example works

Reply:
I think it doesn't work because it is prefilled with "string" and "eutaxonomy-financials".

- "string" won't work because it doesn't match any of the Regexes of ISIN, PermId and LEI
- "eutaxonomy-financials" won't work (yet).  It will work after we fix the eutaxonomyMinusFinancials-problem.
It currently expects you to write eutaxonomyMinusFinancials

Reply:
Yes, also my assumption. The comment was mainly a reminder for me to check that again once the enum subtask is done.

Reply:
It does